backend/lib/models/Notification.js

```diff
@@ -0,0 +1,47 @@
+const { Schema, model } = require("mongoose");
+
+const notificationSchema = new Schema(
+  {
+    action: {
+      enum: ["like", "comment", "post"],
+      required: true,
+      type: String,
+    },
+    post: {
+      id: String,
+      title: String,
+    },
+    receiver: {
+      ref: "User",
+      required: true,
+      type: Schema.Types.ObjectId,
+    },
+    seenAt: Date,
+    sentAt: Date,
+    triggeredBy: {
```

Comment:
Maybe we want the id too? To make a hyperlink to that user's profile

Reply:
Yes, good catch.